samples/performance/subpasses/subpasses.cpp

```diff
@@ -50,6 +50,27 @@ Subpasses::Subpasses()
 	config.insert<vkb::IntSetting>(3, configs[Config::RenderTechnique].value, 0);
 	config.insert<vkb::IntSetting>(3, configs[Config::TransientAttachments].value, 0);
 	config.insert<vkb::IntSetting>(3, configs[Config::GBufferSize].value, 1);
+
+#if defined(PLATFORM__MACOS) && TARGET_OS_IOS && TARGET_OS_SIMULATOR
```

Comment:
This looks like the code above. Can you move this e.g. to a function and put it somewhere central? The iOS simulator is a very niche use case, and this could confuse people.

Reply:
Thanks very much for reviewing this PR quickly.  I understand your concern here as I would prefer not to require  this code either.  I already have an issue logged with the MoltenVK project requesting that Metal Argument Buffers be disabled by default on the iOS Simulator. See https://github.com/KhronosGroup/MoltenVK/issues/2285.

Given the above I have reverted this change for now, and instead will hopefully discuss with @billhollings about the right thing to do re configuring Metal Argument Buffers for the iOS Simulator case.